src/export/InstanceExporter.ts

```diff
@@ -300,6 +300,9 @@ export class InstanceExporter implements Fishable {
     );
     cleanResource(instanceDef);
 
+    // Once all rules are set, we should ensure that we did not add a duplicate profile URL anywhere
+    if (instanceDef.meta?.profile) instanceDef.meta.profile = uniq(instanceDef.meta.profile);
```

Comment:
I wonder if this should be moved into `cleanResource`?  I know the comments for the `cleanResource` function seem to indicate it only removes temporary properties and such, but... it seems like it has evolved since those comments were written.  It looks like it is doing other types of cleanup now too.  What do you think?

Reply:
I'm not entirely against this idea, but my current reasoning for keeping it out of that function is that `cleanResource` is used in other places (okay I think only SD Exporter) but this issue really arises because of how we implement our `InstanceExporter`. The case that this line is trying to fix is really only caused by the addition of `meta.profile` in this exporter. If someone in their Structure Definition had added two of the same profiles, I don't see why that would be right, but I felt like maybe we should avoid un-doing that for them. But then again, there's also an edge case that they could have done that in an Instance as well, which is why I'm don't feel too strongly, I'm just slightly on the `InstanceExporter` side. I could be convinced otherwise.

Reply:
Yeah, OK.  I don't have strong feelings about it, so we'll keep it as-is!